x-pack/test/apm_api_integration/basic/tests/feature_controls.ts

```diff
@@ -126,13 +126,6 @@ export default function featureControlsTests({ getService }: FtrProviderContext)
       expectForbidden: expect403,
       expectResponse: expect200,
     },
-    {
-      req: {
-        url: `/api/apm/services/foo/transactions/charts/throughput?start=${start}&end=${end}&uiFilters=%7B%22environment%22%3A%22testing%22%7D`,
-      },
-      expectForbidden: expect403,
-      expectResponse: expect200,
```

Comment:
Why is this removed? Shouldn't we just add a transaction type?

Reply:
the one above already tests with a transaction type:

https://github.com/elastic/kibana/pull/86893/files/e553e8cccea4ca418164a9af044d6a461afb0f90#diff-00494678ca34f1d18e071f41cc255917ac4027a29f09d27777b5d5b0d940f013L124